_posts/2015-12-16-29.markdown

```diff
@@ -0,0 +1,283 @@
+
+NYC Gov. stuff
+
+\* If you don’t know what a hacker school is, it’s a way to learn programming in a short time by being extremely focused on pragmatic skills.
+
+---- 
+My favourite book this year has been one on human \<-\> computer interactions, [Machines of Loving Grace: The Quest for Common Ground Between Humans and Robots][7]. It help me put a lot of the design decision differences between CocoaPods and Carthage into perspective. It covers a wide range of topics, from self-driving cars to the battle to get Siri treated as an assistant-not just a text to speech engine.
+
+A lot of the books discussion revolves around two types of design paradigms:
+
+* [Intelligence Augmentation][8], coming from [Douglas Engelbart][9], which is mainly focused making it easier for humans to make better decisions.
+* Artificial Intelligence, which is primarily concerned with replacing humans with machines.
+
+There is a lot of space for collaboration between the camps, and often, the book argues that it is a combination of the two that tends to work out best for the people involved. Related: I’ve been reading [The Glass Cage][10], but haven’t got far enough to have an opinion yet.
+
+Some other book highlights:
+
+* [HPMOR][11], a Harry Potter Fan-fiction was finished this year. As a web-serial, I felt like it would never end. I was kind of ok with that too. The podcast version finishes up in 2 weeks too. So it will be good to finish it for a second time. I find the slow pace of audiobooks ensure means I miss less.
+
+* [Worm][12], a web-serial that finished a few years back. As I couldn’t find motivation to do community projects, I opted to re-read some epics. The protagonist in Worm’s impact goes from barely affecting people at high-school to multi-universe mind-melting battles across dimensions. Glad I did a re-read, there’s a lot I understand better on the 2nd run through.
+
+* [Stories of Your Life, and Others][13]. A series of short-stories with some of the best hooks I’ve ever read. “What if there was technology to turn off the ability to be attracted to someone”, “What if the babylonians built a tower and hit the heavens, based on their belief systems at the time”, “What happens when something you took as a core axiom in your belief system was shattered.”
+
+---- 
+
+I’ve lived with [Dave Grandinetti][14] for a year now, this has been amazing. We’ve both changed so much in the last year, in so many subtle ways. After spending so long living in AirBnBs and crashing at Danger’s house, its really satisfying to come back and hang out at a place I can call home.
+
+We have this really cool policy of regularly inviting people to crash at ours, so the place always feels fresh, and it’s nice to have people to interact with. Plus people are much more likely to come over and say hi.
+
+One of the upsides of having a place, is that Danger can actually come over. She NYC visited three times this year. I’ll go into this later.
```

Comment:
I just meant that you wrote "She NYC visited". So I wasn't sure if that was NYC-Visited, or She visited NYC.